.scalafmt.conf

```diff
@@ -0,0 +1,18 @@
+version = 2.3.2
+
+style = default
+
+maxColumn = 100
+
+// Vertical alignment is pretty, but leads to bigger diffs
+align = none
+
+danglingParentheses = false
```

Comment:
These will generate deprecation warnings when scala-steward tries to upgrade this to 2.5.2.  Change to `align.preset` and `danglingParentheses.preset` when that happens.